controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -202,28 +219,33 @@ public void processingFailure(Exception error) {
 
             @Override
             public void connectionDropped() {
+                log.warn("sealSegment {}/{}/{} connectionDropped", scope, stream, segmentNumber);
                 result.completeExceptionally(
                         new WireCommandFailedException(type, WireCommandFailedException.Reason.ConnectionDropped));
             }
 
             @Override
             public void wrongHost(WireCommands.WrongHost wrongHost) {
+                log.warn("sealSegment {}/{}/{} wrongHost", scope, stream, segmentNumber);
                 result.completeExceptionally(
                         new WireCommandFailedException(type, WireCommandFailedException.Reason.UnknownHost));
             }
 
             @Override
             public void segmentSealed(WireCommands.SegmentSealed segmentSealed) {
+                log.info("sealSegment {}/{}/{} segmentSealed", scope, stream, segmentNumber);
                 result.complete(true);
             }
 
             @Override
             public void segmentIsSealed(WireCommands.SegmentIsSealed segmentIsSealed) {
+                log.info("sealSegment {}/{}/{} SegmentIsSealed", scope, stream, segmentNumber);
                 result.complete(true);
             }
 
             @Override
             public void processingFailure(Exception error) {
+                log.error("sealSegment {}/{}/{} failed", scope, stream, segmentNumber, error);
```

Comment:
Same comment as above about adding some info to guide the operator on what to do  in the case this error crops up.